### DISCUSSION TITLE
Installation of Homebrew in mac is failing with error message ambiguous argument 'origin/master': unknown revision or path not in the working tree.

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#3340] [Getting started]
INGUR1M0282:~ hussamoa$ **/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"**
==> Checking for `sudo` access (which may request your password)...
Password:
==> This script will install:
/usr/local/bin/brew
/usr/local/share/doc/homebrew
/usr/local/share/man/man1/brew.1
/usr/local/share/zsh/site-functions/_brew
/usr/local/etc/bash_completion.d/brew
/usr/local/Homebrew

Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /usr/sbin/chown -R hussamoa:admin /usr/local/Homebrew
==> Downloading and installing Homebrew...
**fatal: ambiguous argument 'origin/master': unknown revision or path not in the working tree.**
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
Failed during: git reset --hard origin/master
INGUR1M0282:~ hussamoa$

**Comment by SMillerDev** [comment]
Try the suggestions in #666

**Reply by ghost** [reply]
No it does not work. I doubt if my local git configuration is interfering with the installation.

**Reply by ghost** [reply]
I ran this rm ~/.gitconfig. Then re-installed Homebrew, it worked.

**Reply by lisagorewitdecker** [reply]
same here
